lib/rubocop/cop/util.rb

```diff
@@ -137,18 +137,19 @@ def range_with_surrounding_comma(range, side = :both)
         Parser::Source::Range.new(buffer, begin_pos, end_pos)
       end
 
-      def range_with_surrounding_space(range, side = :both, with_newline = true)
+      def range_with_surrounding_space(range:,
+                                       side: :both,
+                                       newlines: true,
+                                       wspace: false)
```

Comment:
I don't think its worth it to introduce non-existing abbreviations for things to save a few characters. 🙂

Also, if `range` is required and is always provided first anyway, does it need a keyword?

Reply:
> I don't think its worth it to introduce non-existing abbreviations for things to save a few characters. 🙂

Good point.  I'll change this to `whitespace`.

> Also, if range is required and is always provided first anyway, does it need a keyword?

`range` is always required. But what would be the benefit to not using a keyword for it?  My arg for it would be that including `range:` presents a more explicit interface for the method. But I also feel like this sits pretty squarely in grey area. Say the word and I'll change it back 😃 .

Reply:
> But I also feel like this sits pretty squarely in grey area.

Dang. I thought the same thing, and was hoping you could start the conversation with some arguments. 😅 I don't really have any well articulated reasons at the moment. I would use `range:` when I expect to pass the method a hash, where I'm unsure of the order. Otherwise I'd keep it as a normal argument, but I base that on absolutely nothing. 😉

Reply:
The only other argument for using the first required keyword arg instead of a normal arg would be consistency. Instead of the method using 2 types of arguments, it now only employs one type.

---

It's pretty hard to find anything about this online, other than meandering explanations of how to use combinations of argument types.  [Here is the best post I found](https://www.justinweiss.com/articles/fun-with-keyword-arguments/), which made me think about consistency.

Reply:
@Drenmi - Let me know if you ever find any resources that explain why one convention may be better than the other. I'd be interested. This got merged as is, but I still sit in that grey area with no legitimately arguable preference.